gamemodes/terrortown/entities/weapons/weapon_whl_spinner.lua

```diff
@@ -71,7 +71,11 @@ function SWEP:PrimaryAttack()
         local recharge_time = GetConVar("ttt_wheelboy_wheel_recharge_time"):GetInt()
         owner:SetNWInt("WheelBoyNextSpinTime", curTime + recharge_time)
         net.Start("TTT_WheelBoySpinWheel")
-        net.Send(owner)
+        if GetConVar("ttt_wheelboy_reveal_spin"):GetBool() then
+            net.Broadcast()
```

Comment:
Where does the randomization for the wheel spin result occur? If it's on the client we might need to move it to here and network it so that everyone's results are synced

Reply:
Oh, you're right. That happens on the client receive